tgui/packages/tgui/components/StyleableSection.tsx

```diff
@@ -0,0 +1,26 @@
+import { SFC } from 'inferno';
+import { Box } from './Box';
+
+// The cost of flexibility and prettiness.
+export const StyleableSection: SFC<{
+  style?;
+  titleStyle?;
+  textStyle?;
+  title?;
+  titleSubtext?;
```

Comment:
```suggestion
  style?: string,
  titleStyle?: string,
  textStyle?: string,
  title?: string,
  titleSubtext?: string,
```

Reply:
These might not necessarily be strings, so it's best for content holders to be `any` typed, and for style holders to be `Record<string, any>`.